test/e2e/test/elasticsearch/builder.go

```diff
@@ -54,6 +54,8 @@ type Builder struct {
 	// situations where the Elasticsearch resource is modified by an external mechanism, like the autoscaling controller.
 	// In such a situation the actual resources may diverge from what was originally specified in the builder.
 	expectedElasticsearch *esv1.Elasticsearch
+
+	SharedCA bool
```

Comment:
I'm seeing two different naming conventions.  Is it a shared CA, or a global CA?  Or do we want to support both?

Reply:
Good question, I haven't made up my mind yet. But of course a single name for this concept would be preferable.